apps/src/lab2/views/components/Instructions/ResourcePanel/styles.module.scss

```diff
@@ -127,3 +127,8 @@
     }
   }
 }
+
+.navigationFooter {
+  padding: 10px;
+  background-color: var(--background-neutral-secondary)
```

Comment:
Just noticing that this is causing a regression footer area bc the top border is now missing.

On `prod`:
<img width="359" height="256" alt="prod-nav-footer" src="https://github.com/user-attachments/assets/cb681300-9d6e-4479-87c0-2890863483b3" />

On `staging`:

<img width="368" height="250" alt="staging-nav-footer" src="https://github.com/user-attachments/assets/4856b5f7-2cc2-4924-957c-ddd75e6cbe14" />

It looks like if we add `border-top` it would fix and not impact music lab's navigation area view. I think it's okay to keep in `staging` since it's a very small diff. Maybe once resource panel is settled, we can start adding eyes tests.

Reply:
Sorry I didn't catch this earlier!

Reply:
Good catch! Will fix in a follow-up